support-frontend/assets/helpers/internationalisation/country.js

```diff
@@ -421,20 +425,26 @@ export const isInStripePaymentRequestAllowedCountries = (country: IsoCountry) =>
   stripePaymentRequestAllowedCountries.includes(country);
 
 // ----- Functions ----- /
+function sateFromMap(l: string, states: {[p: string]: string}): ?StateProvince {
```

Comment:
should be `stateFromMap`?

Reply:
Whoops, yup will rename to `stateProvinceFromMap`.